README.macOS.md

```diff
@@ -0,0 +1,120 @@
+# Local Install on MacOS with MongoDB
+This will walk through the steps to getting Homebrewery running locally on your computer without needing internet access to create and save brews.  It provides steps on how to get set up with either a **production** environment (if you just want to make brews) or a **development** environment (if you want to submit code changes to project).  Finally, it will provide instruction on how to update to the lastest release over time.  This will be written with no assumption of technical skill as I have very little myself.
+
+## Step 1: Install [Homebrew](https://brew.sh)
+Homebrew is a *package manager* for macOS and Linux, and we will use it to install NodeJS, MongoDB, and git.  You likely will need to be an Administrator on your machine.
```

Comment:
Can you add a couple words to just clarify that Homebrew is not the same as the Homebrewery? Somebody is going to get confused I'm sure.